OpenRA.Mods.Common/Scripting/Global/MediaGlobal.cs

```diff
@@ -168,24 +169,24 @@ public bool PlayMovieInRadar(string movie, LuaFunction playComplete = null)
 		}
 
 		[Desc("Display a text message to the player.")]
-		public void DisplayMessage(string text, string prefix = "Mission")
+		public void DisplayMessage(string text, string prefix = "Mission", bool useCustomColor = false, HSLColor color = new HSLColor())
```

Comment:
Well use https://msdn.microsoft.com/en-us/library/microsoft.visualstudio.modeling.diagrams.hslcolor.white.aspx this then.